packages/frontend/src/components/common/Table/TableProvider.tsx

```diff
@@ -115,24 +112,9 @@ export const TableProvider: FC<React.PropsWithChildren<ProviderProps>> = ({
     }, [stickyColumns, rowColumnWidth]);
 
     const visibleColumns = useMemo(() => {
-        const cols = hideRowNumbers
+        return hideRowNumbers
```

Comment:
so this is sufficient since getFormattedValueCell is called where it needs to be, correct?